File: PHP/uberX.php
<?php
require_once('car.php');
class uberX extends Car{
    public $brand;
    public $model;

    public function __construct($license, $driver, $brand, $model){
        //Acceso a la super clase
        parent::__construct($license, $driver);
        $this->brand = $brand;
        $this->model = $model;
    }

    // Polimorfismo de la impresión de datos
    public function PrintDataCar(){
        echo "License: $this->license
        Name: {$this->driver->name}
        Brand: $this->brand
        Model: $this->model
        ";
    }
}

?>

File: JAVA/UberX.java
public class UberX extends Car{
    // Propiedades propias de la clase UberX
    String brand;
    String model;

    //Constructor que coincide con la clase padre, heredado de la clase car
    public UberX(String license, Account driver, String brand, String model){
        super(license, driver); 
        this.brand = brand;
        this.model = model;
    }  

    //Polimorfismo
    @Override
    void printDataCar() {
        // Haz lo de tu padre
        super.printDataCar();
        // Pero además haz esto
        System.out.println("Brand: " + brand + " model: " + model);
    }
    
}
